docs/content/latest/deploy/multi-dc/async-replication.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Deploy to two data centers with asynchronous replication
+headerTitle: Asynchronous Replication
+linkTitle: Asynchronous Replication
+description: Enable deployment using unidirectional (master-follower) or bidirectional (multi-master) replication between data centers
+menu:
+  latest:
+    parent: multi-dc
+    identifier: async-replication
+    weight: 633
+aliases:
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+You can perform deployment using unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between data centers.
+
+For information on two data center (2DC) deployment architecture and supported replication scenarios, see [Two data center (2DC) deployments](../../../architecture/2dc-deployments).
+
+## Setting Up Universes
+
+You can create producer and consumer universes as follows:
+
+1. Create the yugabyte-producer universe by following the procedure from [Manual deployment](../../manual-deployment).
+1. Create tables for the APIs being used by the producer.
+1. Create the yugabyte-consumer universe by following the procedure from [Manual deployment](../../manual-deployment).
+1. Create tables for the APIs being used by the consumer. These should be the same tables as you created for the producer universe.
+1. Proceed to setting up [unidirectional](#seting-up-unidirectional-replication) or [bidirectional](#setting-up-bidirectional-replication) replication.
+
+## Setting Up Unidirectional Replication
+
+After you've created the required tables, you can set up asysnchronous replication as follows:
+
+- Look up the producer universe UUID and the table IDs for the two tables and the index table on the master UI:
+
+    - To find a universe's UUID in Yugabyte Platform, open the **Universes** tab, click the name of the universe and notice the URL of the universe's **Overview** page that ends with the universe's UUID (for example, `http://myPlatformServer/universes/d73833fc-0812-4a01-98f8-f4f24db76dbe`). 
+    - To find a table's UUID in Yugabyte Platform, click the **Tables** tab on the universe's **Overview** page, then click the table and copy its UUID from its **Overview** tab.
+
```

Comment:
@nspiegelberg @rahuldesirazu hey folks, what do you think makes most sense for us to recommend here, for folks not using the platform?

should we go with the master UI for tables? or yb-admin list_tables?
also, what about the universe UUID? is that a strict requirement to be a UUID? we might be able to use the autogenerated one, but not sure if it shows up in `/varz` as `--cluster_uuid` automatically?

Reply:
I think using `yb-admin list_tables [include_table_id]` would make sense here. And if the --cluster_uuid in /varz wont always have the uuid, maybe we can also check the same field in the cluster config, which I believe should always have it.

Reply:
Changed to:
- To find a universe's UUID, check `/varz` for `--cluster_uuid`. If it is not available in this location, check the same field in the cluster configuration.
- To find a table ID, execute the following command as an admin:
    yb-admin list_tables [include_table_id]